crypto/ex_data.c

```diff
@@ -12,8 +12,9 @@
 
 int do_ex_data_init(OPENSSL_CTX *ctx)
 {
-    OSSL_EX_DATA_GLOBAL *global = openssl_ctx_get_ex_data_global(ctx);
+    OSSL_EX_DATA_GLOBAL *global;
```

Comment:
why did you split this into two lines?

Reply:
I usually prefer to separate declaration from code involving function calls.

If the community prefer to adopt a single way to perform this, I suggest to put it in the coding guidelines to keep it consistent (sometimes is done in a way, sometimes in the other way round).

Reply:
restored the "inline" initialization